doc/howto/move_instances.md

```diff
@@ -54,11 +54,20 @@ For containers, there is limited support.
 Virtual machines can be moved to another server while they are running, thus without any downtime.
 
 To allow for live migration, you must enable support for stateful migration.
-To do so, ensure the following configuration:
+To do so, ensure the following configuration if your instance is not on a shared storage backend:
 
 * Set {config:option}`instance-migration:migration.stateful` to `true` on the instance.
 * Set [`size.state`](devices-disk) of the virtual machine's root disk device to at least the size of the virtual machine's {config:option}`instance-resource-limits:limits.memory` setting.
 
+If you create an instance on a shared storage backend like Ceph RBD, you don't need to set any parameters to enable the live migration. When an instance is created on a shared storage backend, {config:option}`instance-migration:migration.stateful` is set to `true` by default for the instance. Additionally, the [`size.state`](devices-disk) parameter is unset, because it is not required for shared storage.
+
+The default value for {config:option}`instance-migration:migration.stateful` was introduced with LXD 5.21 and applies only to new instances. Existing instances won't be affected by this change.
+
+```{note}
+Some guest OSes (like Centos 8) do not support 9P file system share, which is used for instances that have stateful migration enabled.
```

Comment:
This would be clearer if it expanded on the fact that LXD disables virtiofs filesystem shares when migration.stateful is enabled, and only exports them via 9p protocol. Thus if your guest OS doesn't support 9p the filesystem shares don't work with migration.stateful enabled. We should also mention that this will mean the lxd-agent won't work either.